pkg/network/infraconfigurators/masquerade.go

```diff
@@ -172,17 +172,8 @@ func (b *MasqueradePodNetworkConfigurator) PreparePodNetworkInterface() error {
 }
 
 func (b *MasqueradePodNetworkConfigurator) GenerateDomainIfaceSpec() api.Interface {
-	domainIface := api.Interface{
```

Comment:
You use a dash '-' when you should have used a colon ':' unless you really wanted to say: "link- move GenerateTapDeviceName and GenerateNewBridgedVmiInterfaceName" 

Same for the next commits (I'm aware those 2 buttons are next to each other :) )

Reply:
ping. I think you now changed all the commits to feature a dash instead of a colon in the topic separator of the commits.

Please use colon - ':' .

Reply:
@maiqueb I am sorry but I can't find any restrictions regarding the git commit message format that enforce using a colon instead of a dash.

Reply:
There isn't one. But if you `git log` you'll find plenty of occurrences of separators using colon and not dash, and it costs virtually nothing to preserve consistency.